docs/src/modules/components/AppFrame.js

```diff
@@ -51,6 +51,22 @@ Router.onRouteChangeError = () => {
   NProgress.done();
 };
 
+const AppSearch = React.lazy(() => import('docs/src/modules/components/AppSearch'));
+function DeferredAppSearch() {
+  const fallback = <Skeleton height={32} variant="rect" width={220} />;
```

Comment:
I'm not sure that the Skeleton improves the UX here.

Reply:
> Not sure I like the use of the Skeleton component here.

Reply:
It seems that we agree. It was interesting to try.